README.md

```diff
@@ -75,13 +75,10 @@ const tx = driver.Transaction.makeCreateTransaction(
 const txSigned = driver.Transaction.signTransaction(tx, alice.privateKey);
 
 // send it off to BigchainDB
-driver.Connection
-    .postTransaction(txSigned, API_PATH)
-    .then((res) => {
-        // request the status of the transaction
-        driver.Connection
-            .getStatus(txSigned.id, API_PATH)
-            .then((res) => console.log('Transaction status:', res.status))});
+let conn = driver.Connection(PATH);
```

Comment:
I like this
remind me to upgrade:
- boilerplate
- kyber: examples & tutorials